### DISCUSSION TITLE
Audio del M64Plus Fz Pro

### DISCUSSION THREAD

**Discussion by Roanvil** [mupen64plus-ae/mupen64plus-ae#1132] [Q&A] [answered]
Necesito ayuda, instale el M64Plus Fz tanto de playstore como de Google por apk, pero cuando inicio un juego, el audio no se reproduce y tengo que jugar sin sonido, alguien me ayuda?

**Comment by fzurita** [comment]
Anda a opciones y después audio. Selecciona el modo de bajo rendimiento.

**Reply by Roanvil** [reply, marked as answer]
Gracias, si me funcionó, super
